src/simplewallet/simplewallet.cpp

```diff
@@ -33,7 +33,6 @@
  * 
  * \brief Source file that defines simple_wallet class.
  */
-
```

Comment:
Pedantic, so it's fine if you don't bother, but I dislike random useless changes.